ApplicationPattern+profileInstances.yaml

```diff
@@ -31,10 +31,10 @@ profile-instances:
       label: 'API Documentation'
       display-in-new-browser-window: true
     configuration:
-      protocol: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/protocol'
-      ipv-4-address: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address'
-      port: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port'
-      operation: '/docs'
+      protocol-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/protocol'
+      ipv-4-address-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address'
+      port-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port'
+      operation-reference: '/docs'
```

Comment:
Sorry for the redundant comment, since operation is having two types of values , one is reference(e.g. in line 25) and the other is actual value(e.g. in line 37), can we convert operation attribute to be an object having two sub-attributes like 
operation {
   static-operation: ...
   operation-reference: ...
}

Reply:
Not at all a reason to be sorry. I already recognized the inconsistency late yesterday. Intended to create an issue and solve the problem after closing all currently open pull-requests.   I expect the /docs to be a one and onle exception. So would like to avoid adding an additional field for that. Instead I found it more reasonable to create a OperationServer and to reference it alike in any other case, too.